### DISCUSSION TITLE
Requesting test funds

### DISCUSSION THREAD

**Discussion by pavelsosavel** [entropyxyz/community#133] [Get Test Funds] [answered]
### Address

5Cz9Nu4kst4DhmyF8iJd8e5au61vEgCg4pXhKhcaseU7N46f

**Comment by johnnymatthews** [comment, marked as answer]
Sorry for the delay! Send in block [662,108](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestnet.entropy.xyz#/explorer/query/0x25708bdaa8ab5f6a0e1ade15f2d47e4f6ba59d7dcf2ace97124d5dfa565c8f8e)
